code-of-conduct.html

```diff
@@ -18,59 +18,84 @@
   <body>
     <header>
       <nav>
-        <input type="checkbox" id="nav-toggle" />
-        <label for="nav-toggle" class="burger-menu">
-          <img src="./img/logos/burger.jpg" width="35px" alt="" />
-        </label>
+        <!-- in case of small screens, show a dropdown -->
+        <input type="checkbox" id="nav-toggle" aria-controls="navbar toggle" />
+        <label for="nav-toggle" class="burger-menu"> &#9776;</label>
 
         <ul class="menu">
-          <li><a href="index.html">Home</a></li>
-          <li class="active"><a href="code-of-conduct.html">Code of Conduct</a></li>
-          <li><a href="faq.html">FAQ</a></li>
-          <li><a href="discord.html">Discord</a></li>
+          <li>
+            <a class="nav-logo" href="https://realdevsquad.com/"
+              ><img
+                class="nav-logo-img"
+                src="./img/logos/Real-Dev-Squad@1x.png"
+                alt="Real Dev Squad logo"
+            /></a>
+            <a class="nav-home" href="https://realdevsquad.com/">Home</a>
```

Comment:
As per the discussion, we don't need a `home` button separately. We already are directing the user to our main-site `https://www.realdevsquad.com/` by clicking on RDS logo. Kindly remove the `home` button from the navbar on all pages of welcome site. 😃